src/app/settings/components/Routes/Routes.test.tsx

```diff
@@ -1,154 +1,199 @@
+const dhcpSnippet = dhcpSnippetFactory();
+
+beforeEach(() => {
+  state = rootStateFactory({
+    user: userStateFactory({
+      auth: authStateFactory({ loaded: true, user }),
+      loaded: true,
+      items: [user],
+    }),
+    status: statusStateFactory({ externalAuthURL: null }),
+    licensekeys: licenseKeysStateFactory({ loaded: true, items: [licensekey] }),
+    packagerepository: packageRepositoryStateFactory({
+      loaded: true,
+      items: [packageRepository],
+    }),
+    dhcpsnippet: dhcpSnippetStateFactory({
+      loaded: true,
+      items: [dhcpSnippet],
+    }),
+  });
+});
 
-const mockStore = configureStore();
+const routes = [
+  {
+    title: "General",
+    path: settingsURLs.configuration.general,
+  },
+  {
+    title: "Commissioning",
+    path: settingsURLs.configuration.commissioning,
+  },
+  {
+    title: "Kernel parameters",
+    path: settingsURLs.configuration.kernelParameters,
+  },
+  {
+    title: "Deploy",
+    path: settingsURLs.configuration.deploy,
+  },
+  {
+    title: "Users",
+    path: settingsURLs.users.index,
+  },
+  {
+    title: "Add user",
+    path: settingsURLs.users.add,
+  },
+  {
+    title: `Editing \`${user.username}\``,
+    path: settingsURLs.users.edit({ id: user.id }),
+    pattern: `${settingsURLs.users.edit(null)}/*`,
+  },
+  {
+    title: "License keys",
+    path: settingsURLs.licenseKeys.index,
+  },
+  {
+    title: "Add license key",
+    path: settingsURLs.licenseKeys.add,
+  },
+  {
+    title: "Update license key",
+    path: settingsURLs.licenseKeys.edit({
+      osystem: licensekey.osystem,
+      distro_series: licensekey.distro_series,
+    }),
+    pattern: `${settingsURLs.licenseKeys.edit(null)}/*`,
+  },
+  {
+    title: "Storage",
+    path: settingsURLs.storage,
+  },
+  {
+    title: "Proxy",
+    path: settingsURLs.network.proxy,
+  },
+  {
+    title: "DNS",
+    path: settingsURLs.network.dns,
+  },
+  {
+    title: "NTP",
+    path: settingsURLs.network.ntp,
+  },
+  {
+    title: "Syslog",
+    path: settingsURLs.network.syslog,
+  },
+  {
+    title: "Network discovery",
+    path: settingsURLs.network.networkDiscovery,
+  },
+  {
+    title: "Commissioning scripts",
+    path: settingsURLs.scripts.commissioning.index,
+  },
+  {
+    title: "Upload commissioning script",
+    path: settingsURLs.scripts.commissioning.upload,
+  },
+  {
+    title: "Testing scripts",
+    path: settingsURLs.scripts.testing.index,
+  },
+  {
+    title: "Upload testing script",
+    path: settingsURLs.scripts.testing.upload,
+  },
+  {
+    title: "DHCP snippets",
+    path: settingsURLs.dhcp.index,
+    pattern: settingsURLs.dhcp.index,
+  },
+  {
+    title: "Add DHCP snippet",
+    path: settingsURLs.dhcp.add,
+  },
+  {
+    title: `Editing \`${dhcpSnippet.name}\``,
+    path: settingsURLs.dhcp.edit({ id: dhcpSnippet.id }),
+    pattern: `${settingsURLs.dhcp.edit(null)}/*`,
+  },
+  {
+    title: "Package repos",
+    path: settingsURLs.repositories.index,
```

Comment:
Would you mind changing this to an absolute url import for consistency with other files?

```tsx
import urls from "app/base/urls"
(...)
path: urls.settings.repositories.index,
```